pkg/cmd/server/start/start_node.go

```diff
@@ -495,6 +495,9 @@ func StartNode(nodeConfig configapi.NodeConfig, components *utilflags.ComponentF
 	}
 
 	networkConfig.InternalKubeInformers.Start(wait.NeverStop)
+	if networkConfig.InternalNetworkInformers != nil {
+		networkConfig.InternalNetworkInformers.Start(wait.NeverStop)
```

Comment:
do we need to wait for sync here too?

Reply:
Everything that starts should ALWAYS wait for sync.  We should ensure wait for sync always happens in any controller, because the odds of eventually that causing data loss are too high and there is no downside to waiting for sync.